analytics/utils.py

```diff
@@ -34,6 +34,11 @@ def guess_timezone(dt):
 
     return dt
 
+def remove_slash(host):
```

Comment:
nit: can we rename this to be `remove_trailing_slash`?

Reply:
Agree, thats a bit more descriptive!